### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

#6778
#6772
#6788
#6800
#6770
#6754 
#6753 
#6760 
#6780

### CODE REVIEW

File: src/cloud/docker/docker-containers.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ For example, the following command starts the Docker configuration generator in
 ./vendor/bin/ece-docker build:compose --mode="developer" --php 7.2
 ```
 
-See [See Docker services containters] for details.
+See [Docker service containers] for details.
```
Comment (jeff-matthews) [inline src/cloud/docker/docker-containers.md]: This looks like it's missing something... Shouldn't it be followed by another set of brackets with the name of the link definition like the rest of the links in this topic?

File: src/guides/v2.3/frontend-dev-guide/tools/using_grunt.md [hunk 2]
```diff
@@ -77,6 +77,11 @@ Where the following notation is used:
 -  `<language>`: specified in the `code_subtag` format, for example `en_US`. Only one locale can be specified here. To debug the theme with another locale, create one more theme declaration, having specified another value for `language`.
 -  `<path_to_file>`: path to the root source file, relative to the `app/design/frontend/<Vendor>/<theme>/web` directory. You need to specify all [root source files of the theme]({{ page.baseurl }}/frontend-dev-guide/css-topics/css-preprocess.html#css_preprocess_terms). If your theme [inherits] from a certain theme, and does not contain its own root source files, specify the root source files of the parent theme.
 
+Once these are set correctly, run grunt to watch your changes.
+
+-  Run `grunt exec:<theme>` from the root directory to republish the symlinks.
+-  Run `grunt watch:<theme>`, so that grunt will watch for each and every changes you make.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/tools/using_grunt.md]:
```suggestion
-  Run `grunt watch:<theme>` so that grunt will watch for file changes.
```

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**